frontend/src/bundles/chat/store/slice.ts

```diff
@@ -0,0 +1,39 @@
+import { createSlice, isAnyOf } from '@reduxjs/toolkit';
+
+import { DataStatus } from '~/bundles/common/enums/enums.js';
+import { type ValueOf } from '~/bundles/common/types/types.js';
+
+import { type ChatMessageGetAllItemResponseDto } from '../types/types.js';
+import { createMessage } from './actions.js';
+
+type State = {
+    currentChatMessages: ChatMessageGetAllItemResponseDto[];
+    currentChatId: string | null;
+    dataStatus: ValueOf<typeof DataStatus>;
+};
+
+const initialState: State = {
```

Comment:
we should add info about other chats, to handle sidebar

Reply:
I think it's related to #497